src/components/common/ListNameTranslations.vue

```diff
@@ -41,7 +41,10 @@
           </span>
 
           <!-- more actions menu -->
-          <v-menu offset-y>
+          <v-menu
+            offset-y
+            left
+          >
```

Comment:
I know name translations aren't used for short-form amalgamations, but I fixed it anyway. Example:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/2318e22e-a9d3-4337-8a8e-a886ad694680)